woeip/apps/core/factories.py

```diff
@@ -0,0 +1,19 @@
+import factory
```

Comment:
Since factories are test-related I tend to put them in `tests` directories (e.g. `woeip/apps/core/tests/factories.py`).